pages/administration/productions/[productionSlug]/bookings/[bookingReference]/index.vue

```diff
@@ -15,6 +15,34 @@
         </table-row>
       </table>
     </UiCard>
+    <UiCard title="Ticket Inspection" class="mb-4">
+      <table>
+        <table-row>
+          <table-head-item>Ticket ID</table-head-item>
+          <table-row-item>Seat Group</table-row-item>
+          <table-row-item>Concession Type</table-row-item>
+          <table-row-item>Checked In</table-row-item>
+          <table-row-item v-if="anyTicketsChecked(booking)"
+            >Checked In By</table-row-item
+          >
+          <table-row-item v-if="anyTicketsChecked(booking)"
+            >Checked In At</table-row-item
+          >
+        </table-row>
+        <table-row v-for="ticket in booking.tickets" :key="ticket">
```

Comment:
Key's should typically be scalar (i.e. not a array, class instance or dictionary). I'd recommend
```suggestion
        <table-row v-for="ticket in booking.tickets" :key="ticket.id">
```